src/test/suite/participant/participant.test.ts

```diff
@@ -1279,6 +1281,105 @@ Schema:
           expect(properties.error_name).to.equal('Docs Chatbot API Issue');
         });
       });
+
+      suite('export to playground', function () {
+        beforeEach(async function () {
+          await vscode.commands.executeCommand(
+            'workbench.action.files.newUntitledFile'
+          );
+        });
+
+        afterEach(async function () {
+          await vscode.commands.executeCommand(
+            'workbench.action.closeActiveEditor'
+          );
+        });
+
+        test('exports all code to a playground', async function () {
+          this.timeout(20000);
```

Comment:
Why is this timeout so large? Does the test take a while to run? I'm guessing it's playground opening

Reply:
Left over 👍